queries.sql

```diff
@@ -14,3 +14,98 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg < 10.5;
 SELECT * from animals WHERE name <> 'Gabumon';
 
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- query and update animals tables
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species = '';
+
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT before_weight_update;
+UPDATE animals SET weight_kg = weight_kg * -1;
+SELECT * FROM animals;
+ROLLBACK TO before_weight_update;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
+
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT species, MIN(weight_kg), MAX(weight_kg)
+FROM animals
+GROUP BY species;
+select neutered,sum(escape_attempts) as sum from animals GROUP BY neutered;
+SELECT species, AVG(escape_attempts) AS avg_escape_attempts
+FROM animals
+WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
+GROUP BY species;
+
+
+-- What animals belong to Melody Pond?
+SELECT name 
+FROM animals 
+WHERE owner_id = (
+    SELECT id 
+    FROM owners 
+    WHERE full_name = 'Melody Pond'
+);
```

Comment:
- [ ] Melody Pond has three animals, Chamander, squirtle and blossom but your query from your screenshots only returns two, kindly fix this please.

Reply:
![Screenshot 2023-05-04 231548](https://user-images.githubusercontent.com/109290913/236349691-b954c24d-6f3c-4819-88c9-5caa96c639b0.png)

Reply:
Needs attention!
Screenshots should be updated within the PR description.